### PR TITLE
Refactor: CV page

### PR DESCRIPTION
**Pull request by AlreadyBored**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [x] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

https://github.com/rolling-scopes/rsschool-app/issues/1417

#### 💡 Background and solution

Implemented refactoring of CV page functionality + code structure of corresponding module

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [ ] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: client/src/modules/Opportunities/components/NameTitle/index.tsx [hunk 1]
```diff
@@ -1,19 +1,19 @@
 import React from 'react';
-import { UserData } from '../models';
-import { AvatarCV } from 'modules/Opportunities/components/AvatarCV';
+import { UserData } from 'modules/Opportunities/models';
+import { AvatarCv } from '../AvatarCv';
```
Comment (apalchys) [inline client/src/modules/Opportunities/components/NameTitle/index.tsx:3]:
```suggestion
import { AvatarCv } from '../AvatarCV';
```

File: client/package.json [hunk 2]
```diff
@@ -52,6 +52,7 @@
     "@types/cookie": "0.5.1",
     "@types/jest": "29.0.0",
     "@types/lodash": "4.14.184",
+    "@types/next": "9.0.0",
```
Comment (apalchys) [inline client/package.json:55]: Not sure you need it.

https://www.npmjs.com/package/@types/next

image

**Review by apalchys — APPROVED** [review]
In general, looks good. 

I didn't check every line but if it works I'm fine :)

File: client/package.json (hunk 2 above)
Comment (AlreadyBored) [reply client/package.json:55]: I've got bunch of warnings about lack of types declarations for Next components (e.g. `Header`). If we're okay with this I can remove it :)

Comment (AlreadyBored) [reply client/package.json:55]: hm, it seems that it was minor local issue, removed it

**Pull request merged**